ui/dashboards/src/components/DiscardChangesConfirmationDialog/DiscardChangesConfirmationDialog.tsx

```diff
@@ -11,44 +11,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { IconButton, Dialog, DialogTitle, DialogContent, DialogActions, Button } from '@mui/material';
-import CloseIcon from 'mdi-material-ui/Close';
+import { Button } from '@mui/material';
+import { Dialog } from '@perses-dev/components';
 import { useDiscardChangesConfirmationDialog } from '../../context';
 
-export const DiscardChangesConfirmationDialog = () => {
-  const { discardChangesConfirmationDialog: dialog } = useDiscardChangesConfirmationDialog();
-  const isOpen = dialog !== undefined;
+interface DiscardChangesConfirmationDialogProps {
+  description: string;
+  isOpen: boolean;
+  onCancel: () => void;
+  onDiscardChanges: () => void;
+}
+
+export const DiscardChangesConfirmationDialog = (props: DiscardChangesConfirmationDialogProps) => {
```

Comment:
Basically, everything from the dashboard module that you reuse in another module, should be put in components no ?

Reply:
Yep

Reply:
If it’s just used in dashboards and app, it’s probably fine to stay in dashboards, not a huge deal but moving it should actually be marked in the changelog as a breaking change ( it will require embed users like us at Chronosphere to update the import, we usually use ViewDashboard.tsx, but have one page that pulls in Dashboard.tsx and imports a lot of the dashboards package components separately).

Components package is more for plugin developers to have consistent components and for embed users who want to use an individual component like LineChart / StatChart independent of a dashboard.